build.gradle.kts

```
@@ -64,6 +67,15 @@ subprojects {
                 it.systemProperties["junit.jupiter.execution.parallel.enabled"] = true
                 it.systemProperties["junit.jupiter.execution.parallel.mode.default"] = "concurrent"
             }
+
+            val androidComponentsExtension =
+                extensions.findByName("androidComponents") as AndroidComponentsExtensionImpl<*, *, *>
+            androidComponentsExtension.beforeVariants { builder ->
+                if (builder.name == "playRelease")
```

🤔  shouldn't this be for testing only, not just build name?

```suggestion
                if (rootProject.testReleaseBuild && builder.name == "playRelease")
```

...that is based on my possibly-incorrect understanding that keeper is *only* intended for use in auto-generating keep rules that are important for instrumented testing between the main APK and the test APK ?